auth-service/build.gradle.kts

```diff
@@ -60,19 +60,19 @@ dependencies {
 
     // Metrics
     implementation("io.ktor:ktor-server-metrics-micrometer:$ktorVersion")
-    implementation("io.micrometer:micrometer-registry-cloudwatch2:1.12.4")
+    implementation("io.micrometer:micrometer-registry-cloudwatch2:1.13.0")
 
     //Emails
     implementation("com.sun.mail:jakarta.mail:2.0.1")
 
     // Logging
-    implementation("ch.qos.logback:logback-classic:1.5.3")
+    implementation("ch.qos.logback:logback-classic:1.5.6")
     implementation("net.logstash.logback:logstash-logback-encoder:7.4") // Structured log encoder
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j:1.8.0")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j:1.8.1")
 
     // OpenSAML
-    implementation("org.opensaml:opensaml-core:4.3.0")
-    implementation("org.opensaml:opensaml-saml-impl:4.3.0")
+    implementation("org.opensaml:opensaml-core:4.0.1")
+    implementation("org.opensaml:opensaml-saml-impl:4.0.1")
```

Comment:
This is a downgrade! 4.3.2 is the latest. https://mvnrepository.com/artifact/org.opensaml/opensaml-core?repo=shibboleth-releases

Reply:
Upgraded versions